lib/search/view/search_page.dart

```diff
@@ -0,0 +1,142 @@
+import 'package:flutter/material.dart';
+import 'package:mi_libro_vecino/search/widgets/search_widget.dart';
+import 'package:mi_libro_vecino/ui_utils/colors.dart';
+import 'package:mi_libro_vecino/ui_utils/constans/assets.dart';
+
+class SearchPage extends StatelessWidget {
+  const SearchPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: PColors.purple,
+      appBar: AppBar(
+        leadingWidth: 120,
+        leading: const Padding(
+          padding: EdgeInsets.symmetric(vertical: 22),
+          child: Image(
+            image: AssetImage(Assets.logo),
+          ),
+        ),
+        actions: [
+          TextButton(
+            onPressed: () {},
+            child: Text(
+              'Soy biblioteca',
```

Comment:
We should put this text in the internationalization files. For now we are going to use only Spanish, but its a good practice keep all texts in other file.